src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -149,6 +151,11 @@ internal static string RequiredFeature(this MessageID feature)
         {
```

Comment:
Seems like that should have gone in the original PR for the /features flag. I can throw it in here, but if we're merging this into a features/numeric-literals branch, we probably want that comment in future.